cmd/pubsub/receive_adapter/main.go

```diff
@@ -99,19 +90,22 @@ func mainMetrics(logger *zap.SugaredLogger, opts *metrics.ExporterOptions) {
 		log.Fatalf("Failed to create the metrics exporter: %v", err)
 	}
 
-	// Register the views
-	if err := view.Register(
-		client.LatencyView,
-		transporthttp.LatencyView,
-		json.LatencyView,
-		xml.LatencyView,
-		datacodec.LatencyView,
-		adapter.LatencyView,
-	); err != nil {
-		log.Fatalf("Failed to register views: %v", err)
-	}
-
-	view.SetReportingPeriod(2 * time.Second)
+	// TODO metrics are API surface, so make sure we need to expose this before doing so.
+	//  They seem to be private ones and more profiling related ones.
+	//  Commenting them for now, as we will use pkg stats_reporter.
+	// Register the views.
+	//if err := view.Register(
+	//	client.LatencyView,
+	//	transporthttp.LatencyView,
+	//	json.LatencyView,
+	//	xml.LatencyView,
+	//	datacodec.LatencyView,
+	//  adapter.LatencyView,
```

Comment:
`adapter.LatencyView` is how we get dataplane counts

Reply:
yeah, changed it to use that stats_reporter we are using in other places... 
I put a TODO there to see if we do a refactor later on...